packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/setup-app.md

```diff
@@ -1,15 +1,15 @@
-4. Select either **Native Application** or **Single-Page Application** (depending on the type of application that you are working on) as the **Application type**, then click **Next**.
+1. Select either **Native Application** or **Single-Page Application** (depending on the type of application that you’re building) as the **Application type**, then click **Next**.
 
-    > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have and would break the sign-in or sign-out flow.
+    > **Note:** It's important to choose the appropriate application type for apps that are public clients. Not doing so may result in an Okta API attempting to verify an app's client secret, which public clients aren't designed to have. This would break the app's sign-in and sign-out flows.
 
-5. Specify the **App integration name**.
-6. **Authorization Code** is selected automatically as a **Grant type**. In addition, select **Refresh Token**.
+1. Set the **App integration name**.
+1. Verify that **Authorization Code** is selected as a **Grant type**. In addition, select **Refresh Token**.
 
     > **Note:** For mobile applications, **Authorization Code** is required.
```

Comment:
Auth code is also required for SPA apps? Please confirm with Fed team.

Reply:
Will do